docs/DeveloperGuide.md

```diff
@@ -1,18 +1,616 @@
+
+    For example:
+    * Continuing from example earlier, `dayToRemindUntil = 34`.
+    * `dayToRemindUntil -= NUM_DAYS_IN_OCT`, i.e. 34 - 31
+    * `dayToRemindUntil = 3`, representing that the reminder timeframe extends to 3rd of November
+1. `ReminderListGenerator` then grabs the entries within the reminder timeframe from the list of all recurring entries.
+    * If `isOverflowToNextMonth == true`, it will grab all entries from `currentDay` to `lastDayOfMonth` 
+    and all entries from `1` (1st day of next month) to `dayToRemindUntil`
+    * Else, it will simply grab all entries from `currentDay` to `dayToRemindUntil`
+
+1. Lastly, the list of entries will be converted to a formatted String to be displayed as reminders, and passed back
+to `MenuPrinter`, who will pass it to `UiManager` to print.
+
+The sequence diagram below shows how it works:
+
+![](uml_images/recurringtracker/images/reminderSeqDiagram.png)
+
+
+
+#### Feature 2: FinanceTools
+FinanceTools consists of the following features
+1. Simple Interest Calculator
+2. Yearly/Monthly Compound Interest Calculator
+3. Cashback Calculator
+4. Miles Credit Calculator
+6. Account Storage
+7. Command and Calculation History
+
+##### Simple Interest Calculator
+Simple Interest Calculator is facilitated by ```SimpleIntest``` class. It allows user to calculate interest earned.
+When user inputs ```simple``` as a command, ```handleSimpleInterest``` from ```Handler``` class will handle user
+inputted parameters. The calculation is done by ```SimpleInterest``` class. The result is outputted in
+```FinanceTools.main()```.
+<br />
+
+__Parameters__
+* ```/a``` - Amount (Mandatory)
+* ```/r``` - Interest Rate (Mandatory)
+
+The following class diagram shows how the Simple Interest Calculator feature works:
+<br />
+![ClassDiagram](uml_images/financetools/SimpleInterest/SimpleInterestClassDiagram.png)
+
+The following sequence diagram shows how the params are handled before the implementation is carried out:
+<br />
+![SequenceDiagram1](uml_images/financetools/SimpleInterest/SimpleInterestSequenceDiagram(1).png)
+<br />
+<br />
+The following sequence diagram shows how the Simple Interest Calculator feature works:
+<br />
+![SequenceDiagram2](uml_images/financetools/SimpleInterest/SimpleInterestSequenceDiagram(2).png)
+
+##### Yearly/Monthly Compound Interest Calculator
+Yearly/Monthly Compound Interest Calculator is facilitated by ```YearlyCompoundInterest``` /
+```MonthlyCompoundInterest``` class. It allows user to calculate interest earned.
+When user inputs ```cyearly``` / ```cmonthly``` as a command, ```handleYearlyCompoundInterest``` /
+```handleMonthlyCompoundInterest``` from ```Handler``` class will handle user inputted parameters. The calculation 
+is done by ```YearlyCompoundInterest``` / ```MonthlyCompoundInterest``` class. The result is outputted in
+```FinanceTools.main()```.
+<br />
+
+__Parameters (Yearly Compound Interest Calculator)__
```

Comment:
Perhaps you can consider combining the two similar parts in this section to reduce duplicates.

Reply:
Thank you!